tests/install/steps/test_expansion.py

```diff
@@ -0,0 +1,61 @@
+import pathlib
+import string
+
+from pytest_bdd import scenario, then, parsers, when
+import testinfra
+import kubernetes as k8s
+import yaml
+
+# Scenarios
+@scenario('../features/expansion.feature', 'Add one node to the cluster')
+def test_cluster_expansion(host):
+    pass
+
+# When {{{
+
+@when(parsers.parse('we declare a new node on host "{hostname}"'))
+def declare_node(host, request, kubeconfig, hostname):
```

Comment:
You don't need `host` here:

```suggestion
def declare_node(request, kubeconfig, hostname):
```

Reply:
Done.